src/times.js

```diff
@@ -21,9 +21,13 @@ var _curry2 = require('./internal/_curry2');
  *      R.times(R.identity, 5); //=> [0, 1, 2, 3, 4]
  */
 module.exports = _curry2(function times(fn, n) {
-  var len = Number(n);
-  var list = new Array(len);
-  var idx = 0;
+  var len = Number(n),
+      idx = 0,
+      list;
+  if (len < 0 || isNaN(len)) {
+    throw new RangeError('n must be a non-negative number');
+  }
```

Comment:
hmmm. compromise solution is a `ramda-contrib` package. maybe worth doing? move `times` and `forEach` and maybe some other stuff out of core. idk, sounds like work.